src/DiagnosticsDialog.cpp

```diff
@@ -51,6 +51,12 @@ DiagnosticsDialog::DiagnosticsDialog(QWidget* parent):
     dumpVersionInfo();
 }
 
+class Sleeper : public QThread
```

Comment:
please add a comment so that the next person reading this can figure out why it's even here...